lib/controllers/platform-controller.ts

```diff
@@ -143,6 +143,21 @@ export class PlatformController implements IPlatformController {
 		const hasPlatformDirectory = this.$fs.exists(
 			path.join(projectData.platformsDir, platformName)
 		);
+
+		if (hasPlatformDirectory) {
+			const isPlatformDirectoryEmpty =
+				this.$fs.readDirectory(
+					path.join(projectData.platformsDir, platformName)
+				).length <= 5;
+
+			if (isPlatformDirectoryEmpty) {
+				this.$logger.warn(
+					`${projectData.platformsDir}/${platformName} folder is empty. Run 'ns clean' to generate required files.`,
```

Comment:
```suggestion
					`The platforms/${platformName} folder appears to be invalid. If the build fails, run 'ns clean' and rebuild the app.`,
```

We can omit the full path - reduces noise in the output. Reworded the warning a little, to better reflect what the user should do in case the build fails.